catchpy/settings/base.py

```diff
@@ -15,6 +15,8 @@
 
 from corsheaders.defaults import default_headers
 
+from .aws import get_ecs_task_ips
```

Comment:
See comment below related to `ALLOWED_HOSTS`.

Reply:
Will update the catchpy terraform config in the [atg-ops-appserver](https://github.com/Harvard-ATG/atg-ops-appserver/blob/1c135ae1c5b34124569fa82cad0f5a56649a86e8/terraform/dev/apps/catchpy/main.tf#L19) to include `DJANGO_SETTINGS_MODULE `